lib/chef-dk/skeletons/code_generator/templates/default/kitchen.yml.erb

```diff
@@ -5,11 +5,9 @@ driver:
 provisioner:
   name: chef_zero
 
-# Uncomment the following verifier to leverage Inspec instead of Busser (the
-# default verifier)
-# verifier:
-#   name: inspec
-#   format: doc
+verifier:
+  name: inspec
+  format: doc
```

Comment:
The reason why I ask: We changed the default formatter recently: https://github.com/chef/inspec/pull/794